spec/support/jazz.rb

```diff
@@ -544,11 +544,17 @@ class EnsembleInput < GraphQL::Schema::InputObject
   end
 
   class AddInstrument < GraphQL::Schema::Mutation
+    class << self
+      def prepare_family(value, context)
+        value.upcase
```

Comment:
Slightly concerned that you didn't have to change an expectation anywhere to expect the upcased value?